Configuration/Configuration.py

```diff
@@ -75,3 +55,40 @@
 FIRMWARE_LOAD_ADDRESS = 0x40001000      # Firmware will be loaded at this address when using virtual boot
 FIRMWARE_BINARY_FILE_PATH = "CapuaEnvironment/firmware.bin"
 
+REGISTERS = {
+    "A": 0b0000, "B": 0b0001, "C": 0b0010, "D": 0b0011,
+    "E": 0b0100, "F": 0b0101, "G": 0b0110, "S": 0b0111,
+
+    "A2": 0b1000, "B2": 0b1001, "C2": 0b1010, "D2": 0b1011,
+    "E2": 0b1100, "F2": 0b1101, "G2": 0b1110, "S2": 0b1111
+}
+
+INSTRUCTION_LIST = [
+    "ACTI", "ADD", "AND", "CALL", "CMP", "DACTI", "DIV",
+    "HIRET", "INT", "JMP", "JMPR", "MEMR", "MEMW", "MOV",
+    "MUL", "NOP", "NOT", "NOP","OR", "POP", "PUSH",
+    "RET", "SFSTOR", "SIVR", "SHL", "SHR", "SUB", "XOR"
+]
+
+LABEL_INSTRUCTIONS = [
+    "CALL", "JMP", "MOV", "PUSH"
+]
+
+IDENTIFIER_LIST = [
+    ":", ".GLOBAL", ".DATAALPHA", ".DATANUMERIC", ".DATAMEMREF", ";"
+]
+
+STATE_LIST = {
```

Comment:
If I'm not mistaken, the number in the list is the size of the instruction format. I have two comments here
-As for other assembler related code, I don't feel this belongs here
-The information provided here is the same as the info provided in: spartacus/CapuaEnvironment/IntructionFetchUnit/FormDescription.py
The only difference being the state. Nothing against this but the authority in instruction format should be FormDescription.py.